recipes-containers/docker/docker.inc

```diff
@@ -166,3 +174,7 @@ FILES:${PN}-cli += "${bindir}/docker"
 # NOT rdepend to get a classic one-package install
 DOCKER_UNIFIED_PACKAGE ?= "True"
 RDEPENDS:${PN} += "${@bb.utils.contains("DOCKER_UNIFIED_PACKAGE", "True", "${PN}-cli", "", d)}"
+
+PACKAGES =+ "${PACKAGE_NAME}-buildx-plugin"
```

Comment:
Why ${PACKAGE_NAME} and not ${PN}?

Reply:
${PN} would result in "docker-moby-buildx-plugin"
${PACKAGE_NAME} would result in "docker-buildx-plugin"

According to https://docs.docker.com/engine/install/centos/#installation-methods (referenced in the above comment) the buildx-plugin is not intended to have "moby" or "ce" within it's package name